src/test/scala/is/hail/variant/vsm/VSMSuite.scala

```diff
@@ -199,6 +198,13 @@ class VSMSuite extends SparkSuite {
 
     p.check()
   }
+  
+  @Test def testWriteReadFile() {
```

Comment:
?  This duplicates the functionality of a number of tests in `ExportVCFSuite`, `ImportVCFSuite`.

Reply:
Removed. I'd added when I was getting the mysterious difference in behavior but you're right this is also tested elsewhere.